examples/text_to_image/train_text_to_image.py

```diff
@@ -407,6 +420,24 @@ def parse_args():
     return args
 
 
+def get_save_iteration(output_dir):
+    checkpoints_dir = os.path.join(output_dir, "checkpoints")
+
+    if not os.path.isdir(checkpoints_dir):
+        return 0
+
+    dirs = os.listdir(checkpoints_dir)
+    dirs = [d for d in dirs if d.startswith("checkpoint")]
+
+    save_iterations = [int(d.split("_")[1]) for d in dirs]
+    save_iterations.sort()
+
+    save_iteration = save_iterations[-1]
+    save_iteration += 1
+
+    return save_iteration
```

Comment:
accelerate's automatic checkpoint naming will default to saving from "checkpoint zero" every time. Accelerate also doesn't overwrite existing checkpoints, and instead throws an error. We now have to tell accelerate which checkpoint to start from